src/material-examples/table-http/table-http-example.ts

```diff
@@ -48,69 +75,15 @@ export interface GithubIssue {
 }
 
 /** An example database that the data source uses to retrieve data for the table. */
-export class ExampleHttpDataSource {
-  constructor(private _httpClient: HttpClient) {}
+export class ExampleHttpDao {
+  constructor(private http: HttpClient) {}
 
   getRepoIssues(sort: string, order: string, page: number): Observable<GithubApi> {
     const href = 'https://api.github.com/search/issues';
     const requestUrl =
-      `${href}?q=repo:angular/material2&sort=${sort}&order=${order}&page=${page + 1}`;
+        `${href}?q=repo:angular/material2&sort=${sort}&order=${order}&page=${page + 1}`;
 
-    return this._httpClient.get<GithubApi>(requestUrl);
+    return this.http.get(requestUrl)
+        .map(response => response as GithubApi);
```

Comment:
Maybe it's nicer to use the type from HttpClient? `this.http.get<GithubApi>(requestUrl)`